KarhooUISDK/Screens/PassengerDetails/PassengerDetailsViewController.swift

```diff
@@ -58,6 +58,12 @@ final class PassengerDetailsViewController: UIViewController, PassengerDetailsVi
         }
     }
     
+    var focusPhoneNumber: Bool = true {
```

Comment:
Is adding a new variable really required? This is very case-by-case approach and should be avoided. 
Find the place in presenter where it is informed about edition change, compare result (if new value is provided) and if it is, focus next input view.